src/ReviewRequest/list/hooks/apis/useQueryPrReviews.ts

```diff
@@ -0,0 +1,15 @@
+import type { ApiResponse } from "@/common/types/api";
+import { useQuery } from "@tanstack/react-query";
+import {
+  type QueryPrReviewsQueryParams,
+  type QueryPrReviewsResponse,
+  queryPrReviews,
+} from "../../apis/query-pr-reviews";
+import { prListQueryKeys } from "./querykeys";
+
+export const useQueryPrReviews = (params?: QueryPrReviewsQueryParams) => {
+  return useQuery<ApiResponse<QueryPrReviewsResponse>, Error>({
```

Comment:
```suggestion
  return useQuery({
```

요기도 추론돼서 없어도 괜찮을 것 같아욤